build/metadata/bitbucket/README.md

```diff
@@ -5,7 +5,7 @@ Job for finding and sending feature flag code references to LaunchDarkly
 Add the following snippet to the script section of your `bitbucket-pipelines.yml` file:
 
 ```yaml
-- pipe: - pipe: launchdarkly/ld-find-code-refs-pipe:2.4.0
```

Comment:
Replacing actual versions here with x.x.x so devs will immediately realise that these numbers will be auto-updated to prevent manual editing.